package/network/services/lldpd/files/lldpd.init

```diff
@@ -349,34 +375,36 @@ reload_service() {
 		return 0
 	fi
 
-	$LLDPCLI -u "$LLDPSOCKET" 2>&1 /dev/null <<-EOF
+	$LLDPCLI -u "$LLDPSOCKET" >/dev/null 2>&1 <<-EOF
 		pause
 		unconfigure lldp custom-tlv
+		unconfigure lldp capabilities-advertisements
+		unconfigure lldp management-addresses-advertisements
 		unconfigure system interface pattern
 		unconfigure system description
 		unconfigure system hostname
 		unconfigure system ip management pattern
 		unconfigure system platform
 	EOF
 	if [ "$CONFIG_LLDPD_WITH_LLDPMED" = "y" ]; then
-		$LLDPCLI -u "$LLDPSOCKET" 2>&1 /dev/null <<-EOF
+		$LLDPCLI -u "$LLDPSOCKET" >/dev/null 2>&1 <<-EOF
 			unconfigure med fast-start
 		EOF
 
 	fi
 	# Rewrite lldpd.conf
 	# If something changed it should be included by the lldpcli call
 	write_lldpd_conf
-	$LLDPCLI -u "$LLDPSOCKET" -c "$LLDPD_CONF" -c "$LLDPD_CONFS_DIR" 2>&1 /dev/null
+	$LLDPCLI -u "$LLDPSOCKET" -c "$LLDPD_CONF" -c "$LLDPD_CONFS_DIR" >/dev/null 2>&1 
 	# Broadcast update over the wire
-	$LLDPCLI -u "$LLDPSOCKET" 2>&1 /dev/null <<-EOF
+	$LLDPCLI -u "$LLDPSOCKET" >/dev/null 2>&1 <<-EOF
 		resume
 		update
 	EOF
 	return 0
 }
 
 stop_service() {
-	rm -rf ${LLDPD_RUN} "$LLDPSOCKET" 2> /dev/null
+	rm -rf ${LLDPD_RUN} "$LLDPSOCKET" >/dev/null 2>&1
```

Comment:
What message is this going to generate such that `stderr` needs to be discarded?

Reply:
It's possible to build lldpd without socket support. References to the socket are discarding errors throughout the script, presumably to allow for builds without the socket support.

Reply:
Or, in case no service was actually running when someone runs 'stop'.

Reply:
That's fine, but removing a non-existent file with `rm -rf` won't generate any output anyway...

```
       -f, --force
              ignore nonexistent files and arguments, never prompt
```

Reply:
That's true. The spurious output at the moment needs fixing, however, so the parameter order must be changed.